internal/pkg/deploy/cloudformation/stack/env_integration_test.go

```diff
@@ -270,7 +270,8 @@ network:
       private:
         - id: 'subnet-33333'
         - id: 'subnet-44444'
-    flow_logs: on`
+    flow_logs:
+     retention: 60`
```

Comment:
:rip: I kinda wish we made the unit more explicit.
Like accepting `60d` as input or `60`.